website/client/components/groups/chatTextarea.vue

```diff
@@ -0,0 +1,223 @@
+<template lang="pug">
+  .row.chat-row
+    .col-12
+      h3(v-once) {{ label }}
+
+      .row
+        textarea(:placeholder='placeholder',
+                  v-model='newMessage',
+                  :class='{"user-entry": newMessage}',
+                  @keydown='updateCarretPosition',
+                  @keyup.ctrl.enter='sendMessageShortcut()',
+                  @paste='disableMessageSendShortcut()'
+                )
+        autocomplete(
+                :text='newMessage',
+                v-on:select="selectedAutocomplete",
+                :coords='coords',
+                :chat='group.chat')
+
+      .row.chat-actions
+        .col-6.chat-receive-actions
+          button.btn.btn-secondary.float-left.fetch(v-once, @click='fetchRecentMessages()') {{ $t('fetchRecentMessages') }}
+          button.btn.btn-secondary.float-left(v-once, @click='reverseChat()') {{ $t('reverseChat') }}
+        .col-6.chat-send-actions
+          button.btn.btn-secondary.send-chat.float-right(v-once, @click='sendMessage()') {{ $t('send') }}
+
+      community-guidelines
+
+      slot(
+        name="additionRow",
+      )
+
+      .row
+        .hr.col-12
+        chat-message(:chat.sync='group.chat', :group-id='group._id', :group-name='group.name')
```

Comment:
yeah I didn't think we'd need add chat-message as a slot on both uses, hmm any name suggestions ? `chat` since its both the new message and the chatlist itself

Reply:
chat sounds good to me